### ISSUE TITLE
CSS comment spacing disregards `newline_between_rules`, `selector_separator_newline`

### ISSUE THREAD

**Issue by jawshooah** [beautify-web/js-beautify#645]
Original:

``` css
/* Comment above first rule */

body {
    display: none;
}

/* Comment between rules */

ul,

/* Comment between selectors */

li {
    display: none;
}

/* Comment after last rule */
```

Regardless of the values of `newline_between_rules` and `selector_separator_newline`, the output is the same:

``` css
/* Comment above first rule */

body {
    display: none;
}
/* Comment between rules */

ul,
/* Comment between selectors */

li {
    display: none;
}
/* Comment after last rule */
```

**Comment by anephew** [comment]
Fixed?

**Comment by bitwiseman** [comment]
Still open, so not fixed.

**Comment by MacKLess** [comment]
This appears to have been resolved as of v1.8.0-rc13. @jawshooah  and @anephew, if this doesn't appear to be working correctly, can you please let me know.
